connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/utils/ProtobufUtils.scala

```diff
@@ -223,28 +226,28 @@ private[sql] object ProtobufUtils extends Logging {
     }
   }
 
-  private def parseFileDescriptorSet(descFilePath: String): List[Descriptors.FileDescriptor] = {
-    var fileDescriptorSet: DescriptorProtos.FileDescriptorSet = null
+  def readDescriptorFileContent(filePath: String): Array[Byte] = {
```

Comment:
This function is duplicate of the one in connect client. Is there way to define them in a common place? I think this is a general question of connect. No need to address in this PR.

Reply:
Actually, there is already a `common-utils` module in Spark and @amaliujia is working on it. I think connect client will also rely on the `common-utils` module In the future, then we can put these them in that module.